src/output.rs

```diff
@@ -3,28 +3,35 @@ use std::io::{self, Write};
 use std::path::Path;
 
 use lscolors::{Indicator, LsColors, Style};
+use once_cell::sync::Lazy;
 
 use crate::config::Config;
+use crate::dir_entry::DirEntry;
 use crate::error::print_error;
 use crate::exit_codes::ExitCode;
 use crate::filesystem::strip_current_dir;
 
+static MAIN_SEPARATOR_STR: Lazy<String> = Lazy::new(|| std::path::MAIN_SEPARATOR.to_string());
```

Comment:
should this use `filesystem::default_path_separator instead of `std::path::MAIN_SEPARATOR`?

Reply:
oh, nevermind, in that case `path_separator` should be set to a `Some`.